text_generation/causal_lm/cpp/continuous_batching/library/src/sequence_group.hpp

```diff
@@ -91,14 +91,14 @@ class Sequence {
 
     // appends new tokens to a generated part
     void append_token(int64_t token_id, float log_prob) {
-        m_cumulative_log_prob += log_prob;
+        m_log_probs.push_back(log_prob);
         m_generated_ids.push_back(token_id);     
     }
 
-    GenerationOutput get_last_generation_output(const GenerationConfig& sampling_params) {
+    GenerationOutput get_last_generation_output() {
         GenerationOutput output;
-        output.score = get_beam_search_score(sampling_params);
```

Comment:
do we need to fix similar thing in `get_finished_sequences` ? to sort them based on logprobs instead of beam score for non-beam search sampling.

Reply:
Moved to open in JIRA. Should be done in the next PRs